components/button/button.css

```diff
@@ -0,0 +1,207 @@
+.buttons-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 80%;
+  padding: 2rem;
+  margin: 1rem auto;
+  background-color: var(--comp-container-color);
+  gap: 1rem;
+}
+
+/* Button */
+.btn {
+  border: none;
+  background: transparent;
+  font-size: 1rem;
+  font-family: "PT Sans Narrow", sans-serif;
+  color: var(--ghost-color);
+}
+
+.btn:hover {
+  cursor: pointer;
+}
+
+/* Buttons with Brand Colors & Buttons with Active State & Buttons with an Outline & Buttons Disabled & Buttons with State Colors */
+.btn-brand-colors > button,
+.btn-active-state > button,
+.btn-outline > button,
+.btn-disabled > button,
+.btn-state-colors > button {
+  text-transform: uppercase;
+  padding: 0.5rem 1rem;
+}
+
+.btn-brand-colors > .btn-neutral,
+.btn-active-state > .btn-neutral {
+  background-color: var(--neutral-color);
+}
+
+.btn-brand-colors > .btn-primary,
+.btn-active-state > .btn-primary {
+  background-color: var(--primary-color);
+}
+
+.btn-brand-colors > .btn-secondary,
+.btn-active-state > .btn-secondary {
+  background-color: var(--secondary-color);
+}
+
+.btn-brand-colors > .btn-accent,
+.btn-active-state > .btn-accent {
+  background-color: var(--accent-color);
+}
+
+.btn-brand-colors > .btn-ghost,
+.btn-active-state > .btn-ghost {
+  color: var(--ghost-color);
+}
+
+.btn-brand-colors > .btn-link > a,
+.btn-active-state > .btn-link > a {
+  text-decoration: none;
+}
+
+/* Buttons with Brand Colors on Hover*/
+
+/* Buttons with an Outline */
+.btn-outline > button.btn-neutral {
+  border: 1px solid var(--neutral-color);
+  color: var(--neutral-color);
+}
+
+.btn-outline > button.btn-primary {
+  border: 1px solid var(--primary-color);
+  color: var(--primary-color);
+}
+
+.btn-outline > button.btn-secondary {
+  border: 1px solid var(--secondary-color);
+  color: var(--secondary-color);
+}
+.btn-outline > button.btn-accent {
+  border: 1px solid var(--accent-color);
+  color: var(--accent-color);
+}
+
+/* Buttons with an Outline on Hover */
+
+/* Buttons Disabled */
+.btn-disabled > button {
+  background-color: var(--neutral-color);
+}
+
+/* Buttons Disabled on Hover*/
+.btn-disabled > button:hover {
+  cursor: not-allowed;
+}
+
+/* Buttons with State Colors */
+.btn-state-colors > button.btn-info {
+  background-color: var(--info-color);
+  color: var(--ghost-color);
+}
+
+.btn-state-colors > button.btn-success {
+  background-color: var(--success-color);
+  color: var(--ghost-color);
+}
+
+.btn-state-colors > button.btn-warning {
+  background-color: var(--warning-color);
+  color: var(--ghost-color);
+}
+.btn-state-colors > button.btn-danger {
+  background-color: var(--danger-color);
+  color: var(--ghost-color);
+}
+
+/* Circular Buttons with different Sizes */
+.btn-size {
+  padding: 1rem;
+}
+
+.circle > button {
+  border-radius: 50%;
+}
```

Comment:
Well, I don't know if it's intensional, but, the circular buttons are not circular on themselves, and are circular because of this hierarchical declaration here, i.e. only the buttons inside a div with this `circle` class will be circular. I am not sure if it's a good practice, since every time I would want a circular button I would have to put it inside a div with a class `circle`.

Reply:
Yeah, I made the required changes.